### ISSUE TITLE
While tackling MOTD (ubuntu-notifier) let us also consider tuning MOTD (esm announce)

### ISSUE THREAD

**Issue by cpaelzer** [canonical/ubuntu-pro-client#2533]
Issue #2458 and [discourse discussions](https://discourse.ubuntu.com/t/remove-pro-advertising-from-apt/34051) have asked for MOTD message disabling in regard to https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/explanations/motd_messages.html

But there is actually more MOTDing:

```
$ dpkg -S /etc/update-motd.d/88-esm-announce
ubuntu-advantage-tools: /etc/update-motd.d/88-esm-announce
```

provides

```
$ cat /var/lib/ubuntu-advantage/messages/motd-esm-announce

 * Introducing Expanded Security Maintenance for Applications.
   Receive updates to over 25,000 software packages with your
   Ubuntu Pro subscription. Free for personal use.

     https://ubuntu.com/pro
```

This request is two-fold (feel free to split if one is particularly faster to solve than the other).
1. searches [1](https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/search.html?q=88-esm-announce&check_keywords=yes&area=default) [2](https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/search.html?q=esm-announce&check_keywords=yes&area=default#) [3](https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/search.html?q=announce&check_keywords=yes&area=default#) didn't return anything useful - this part of the motd interactions should also be documented
2. when we get to tackle making disabling motd integration less burdensome we should make it disable not only disable those of apt-notifier and `91-contract-ua-esm-status`, but right away also cover that of `88-esm-announce`

**Comment by cpaelzer** [comment]
I've made up my mind since we already remove it in 27.14
But we should document it well for people still finding it in logs and old posts.

**Comment by cpaelzer** [comment]
FYI - to be fixed by #2534
